### PR TITLE
Preserve line endings that libsass hands us

### LINKED ISSUE #160: Consistent line endings between platforms
We're using libsass-python to compile SCSS on different platforms. Unfortunately, on Windows systems the generated CSS contains windows line endings while on Linux systems, Linux line endings are used.

This creates a problem where unnecessary whitespace changes make reviews harder.

I would suggest either using one set of endings consistently (Linux, for example) OR analyzing the file endings of the input file and sticking with those.

### PR DESCRIPTION
**Pull request by asottile**
Resolves #160

### CODE REVIEW

File: sasstests.py [hunk 1]
```diff
@@ -749,7 +744,7 @@ def test_sassc_output(self):
             assert exit_code == 0
             assert self.err.getvalue() == ''
             assert self.out.getvalue() == ''
-            with open(tmp) as f:
+            with io.open(tmp, encoding='UTF-8', newline='') as f:
```
Comment (asottile) [inline sasstests.py]: this acts as a regression test

**Comment by asottile** [comment]
@dahlia thoughts on this?  It changes behaviour but makes our wrapper both more transparent and more like the other implementations.

File: sasstests.py [hunk 2]
```diff
@@ -643,25 +643,20 @@ def test_wsgi_sass_middleware(self):
...
-            self.assert_bytes_equal(b'/asdf', r.data)
+            self.assertEqual(b'/asdf', r.data)
             assert r.mimetype == 'text/plain'
             r = client.get('/static/a.scss.css')
             assert r.status_code == 200
-            self.assert_bytes_equal(
+            self.assertEqual(
                 b(A_EXPECTED_CSS_WITH_MAP),
-                r.data
+                r.data,
```
Comment (frostbtn) [inline sasstests.py]: a comma of imperfection :)

Comment (asottile) [inline sasstests.py]: Trailing commas are a best practices :D

**Comment by dahlia** [comment]
Looks good to me!

**Comment by asottile** [comment]
<3

**Pull request merged**